builtin/providers/azurerm/resource_arm_virtual_network.go

```diff
@@ -45,6 +45,10 @@ func resourceArmVirtualNetwork() *schema.Resource {
 				Required: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"id": &schema.Schema{
```

Comment:
@phinze, @jen20: note that I added tracking for the ID for inner-vn-subnets for referencing as well...